packages/node-cli/src/metadata/SdkCommandsMetadataPatch.json

```diff
@@ -0,0 +1,24 @@
+{
+	"account:manageauth": {
+		"options": {
+			"info": {
+				"description": "Prints the following information for the specified authentication ID (authID): account ID, role ID, and url. Usage: account:manageauth --info {authID}."
+			},
+			"list": {
+				"description": "Prints a list of all the configured authentication IDs (authID). Usage: account:manageauth --list."
+			},
+			"remove": {
+				"description": "Remove an authentication ID (authID). Usage: account:manageauth --remove."
+			},
+			"rename": {
+				"description": "Rename an authentication ID (authID). You must specify it together with the renameto option. Usage: account:manageauth -rename {authID} -renameto {newauthID}."
+			},
+			"renameto": {
+				"description": "Enter the new ID for an authentication ID (authID). You must specify it together with the rename option. Usage: account:manageauth -rename {authID} -renameto {newauthID}."
+			},
```

Comment:
We need to change the `-rename` and `-renameto` options appearances for the double dash nodeCLI `--rename` and `--renameto`.